release.py

```diff
@@ -0,0 +1,289 @@
+            )
+            record_step("install_deps_on_venv_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to install virtualenv and and dependencies clone! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    if step_complete("branch_complete"):
+        print(f"{bcolors.OKCYAN}Skipping create release branch...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(["git", "branch", BRANCH], check=True)
+            record_step("branch_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to clean release branch! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    subprocess.run(["git", "checkout", BRANCH])
+    if step_complete("tox_news_complete"):
+        print(f"{bcolors.OKCYAN}Skipping news generation...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(["./venv/bin/tox", "-e", "news"], check=True)
+            record_step("tox_news_complete")
+        except Exception as e:
+            print(f"{bcolors.FAIL}INFO: Failed to generate news! :(...{bcolors.ENDC}")
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    if step_complete("newsfragment_complete"):
+        print(f"{bcolors.OKCYAN}Skipping add news fragment...{bcolors.ENDC}")
+    else:
+        try:
+            mkfile("newsfragments/{ticket}.minor".format(ticket=TICKET))
+            record_step("newsfragment_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to add newsfragment file! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    if step_complete("commit_newsfragment_complete"):
+        print(f"{bcolors.OKCYAN}Skipping commit fragment...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(["git", "add", "."], check=True)
+            subprocess.run(
+                ["git", "commit", "-s", "-m", "tahoe-lafs-{tag} news".format(tag=TAG)],
+                check=True,
+            )
+            record_step("commit_newsfragment_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to commit newsfragment! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    updated_content = None
+    LINE = "=" * len(RELEASE_TITLE)
+    with open("NEWS.rst", "r") as f:
+        content = f.read()
+        updated_content = re.sub(
+            r"\.\.\stowncrier start line\n(Release\s(\d+\.\d+\.\d)(\.)post\d+\s\(\d{4}-\d{02}-\d{02}\))+(\n\'+)",
+            RELEASE_TITLE + LINE,
+            content,
+        )
+    with open("updated_news.rst", "w") as f:
+        f.write(updated_content)
+    shutil.move("updated_news.rst", "NEWS.rst")
+    print(f"{bcolors.OKGREEN}First set of release steps completed! :).{bcolors.ENDC}")
```

Comment:
These instructions should say _where_ the release happened (because e.g. checking `NEWS.rst` will be confusing, since this didn't "do the release" in this repository, it made a new directory).